src/app/dog/dog.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Params, Router } from '@angular/router';
+import { DogsService } from '../dogs/dogs.service';
+
+@Component({
+	selector: 'app-dog',
+	templateUrl: './dog.component.html',
+	styleUrls: ['./dog.component.css']
+})
+export class DogComponent implements OnInit {
+
+	public subBreeds: any[] = [];
```

Comment:
specify a data type for this array. You can define an interface SubBreed if you have more than one property